client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -962,6 +966,35 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 		/>
 	);
 
+	const bigSkyButtonEventProperties = {
+		is_big_sky_eligible: isBigSkyEligible,
+		// is_filter_included_with_plan_enabled: true/false,
+		// preselected_filters: ??,
+		// selected_filters: ??,
+		// {filter} ??
+	};
+	const bigSkyButtons = (
+		<>
+			{ isBigSkyEligible && (
+				<Button
+					href={ `/setup/site-setup/launch-big-sky?siteSlug=${ siteSlug }&siteId=${ site.ID }` }
```

Comment:
I suppose we can simply use `navigate( 'launch-big-sky' )` to go to the big sky step 🙂

Reply:
`navigate('launch-big-sky')` is ignoring query params and making the route invalid. 
Used complete path, now it doesn't cause complete app reload. 👍 
`navigate( `/setup/site-setup/launch-big-sky?siteSlug=${ siteSlug }&siteId=${ site.ID }` );`